.github/workflows/main.yaml

```diff
@@ -11,12 +11,26 @@ on:
       - 'docs/**'
     branches:
       - '*'
+  workflow_call:
+    inputs:
+      build_multiarch:
+        required: true
+        type: boolean
 
 jobs:
   test-containers:
     runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v3
+    
+    - name: Free up disk space
+      uses: jlumbroso/free-disk-space@main
+      with:
+        android: true
+        dotnet: true
+        haskell: true
+        large-packages: true
+        swap-storage: true
```

Comment:
I think we should use a condition to use this action only if `build_multiarch` is `true`